docs/transpile/ai-transpiler-passes.mdx

```diff
@@ -51,7 +51,6 @@ The `layout_mode` includes the following options:
 - `improve`: This uses the layout set by the previous transpiler passes as a starting point. It is useful when you have a good initial guess for the layout; for example, for circuits that are built in a way that approximately follows the device's coupling map. It is also useful if you want to try other specific layout passes combined with the `AIRouting` pass.
 - `optimize`: This is the default mode. It works best for general circuits where you might not have good layout guesses. This mode ignores previous layout selections.
 
-<Admonition type="tip" title="Recommendations">
-    - Learn [how to transpile circuits](https://learning.quantum.ibm.com/tutorial/submit-transpiled-circuits) as part of Qiskit Patterns workflows using Qiskit Runtime.
```

Comment:
This link still seems to work -- or did I miss something about its eventual removal?

Reply:
My mistake!